scripts/wait_beaker_dataset_model_upload_then_evaluate_model.py

```diff
@@ -25,8 +25,6 @@ class Args:
 
 def main(args: Args, beaker_runtime_config: BeakerRuntimeConfig):
     print(args)
-    beaker_dataset_ids = get_beaker_dataset_ids(beaker_runtime_config.beaker_workload_id, sort=True)
-    print(beaker_experiment_succeeded(beaker_runtime_config.beaker_workload_id))
```

Comment:
We also don't need these runs. They are prob the culprit for `'0001-01-01T00:00:00Z'` because the job was still running.